internal/hud/server/logs_reader.go

```diff
@@ -28,20 +28,22 @@ type WebsocketReader struct {
 	conn         WebsocketConn
 	marshaller   jsonpb.Marshaler
 	unmarshaller jsonpb.Unmarshaler
+	keepAlive    bool // whether to keep listening on websocket, or close after first message
```

Comment:
fwiw, i would probably not call this keepAlive, because that's a pretty common term of art in this space related to the mechanism for keeping connections open, and you'll tend to see it a lot in networking code

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Keep-Alive
https://pkg.go.dev/google.golang.org/grpc@v1.31.0?tab=doc#WithKeepaliveParams

Reply:
ah yep -- I remembered that this term was already A Thing but now how _specific_ a thing it was.